src/main/java/com/mraof/minestuck/block/redstone/RemoteObserverBlock.java

```diff
@@ -0,0 +1,100 @@
+package com.mraof.minestuck.block.redstone;
+
+import com.mraof.minestuck.client.gui.MSScreenFactories;
+import com.mraof.minestuck.effects.CreativeShockEffect;
+import com.mraof.minestuck.tileentity.redstone.RemoteObserverTileEntity;
+import com.mraof.minestuck.util.ParticlesAroundSolidBlock;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.particles.RedstoneParticleData;
+import net.minecraft.state.BooleanProperty;
+import net.minecraft.state.StateContainer;
+import net.minecraft.state.properties.BlockStateProperties;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.*;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.BlockRayTraceResult;
+import net.minecraft.world.IBlockReader;
+import net.minecraft.world.World;
+
+import javax.annotation.Nullable;
+import java.util.Random;
+
+/**
+ * Gives off power if the boolean condition currently set in the tile entity is met. Checks for an entity meeting its condition in a 16 block radius.
+ * GUI is limited by creative shock
+ */
+public class RemoteObserverBlock extends Block
+{
+	public static final BooleanProperty POWERED = BlockStateProperties.POWERED;
+	
+	public RemoteObserverBlock(Properties properties)
+	{
+		super(properties);
+		registerDefaultState(stateDefinition.any().setValue(POWERED, false));
+	}
+	
+	@Override
+	@SuppressWarnings("deprecation")
+	public ActionResultType use(BlockState state, World worldIn, BlockPos pos, PlayerEntity player, Hand hand, BlockRayTraceResult hit)
+	{
+		if(!player.isCrouching() && !CreativeShockEffect.doesCreativeShockLimit(player, 1))
+		{
+			TileEntity tileEntity = worldIn.getBlockEntity(pos);
+			if(tileEntity instanceof RemoteObserverTileEntity)
+			{
+				RemoteObserverTileEntity te = (RemoteObserverTileEntity) tileEntity;
+				
+				MSScreenFactories.displayRemoteObserverScreen(te);
+			}
+		}
+		
+		return ActionResultType.SUCCESS;
```

Comment:
Doesn't seem right to always return the same result type. See the comment in the stat storer block.